app/scripts/modules/core/src/domain/IPipeline.ts

```diff
@@ -28,13 +28,15 @@ export interface IPipeline {
     type: string;
   };
   type?: string;
+  pinAllParameters?: boolean;
```

Comment:
there's an outstanding conversation on this line from a prior PR:
https://github.com/armory-io/deck/pull/4#discussion_r270636672
<kbd>![](https://cl.ly/b91f0f7c63cd/download/Screen%20Shot%202019-04-01%20at%2017.36.10.png)</kbd>

Reply:
I'm good with removing this if we decide `toggle all pins`/`unpin all`+`pin all` is the best option.